pesviewer/pesviewer.py

```diff
@@ -910,9 +910,50 @@ def get_smis(m, smis, files):
                     print('Could not generate smiles for {n}'.format(n=m.name))
                 # end try
             # end for
-            return smis
+        return(smis)
     # end def
 
+    def reaction_smi():
+        """
+        Given a list of transition states and barrierless channels
+        write a reaction_smi.out file that contains the reactions as, e.g.:
+        OOC[CH2] = [OH] + O1CC1i  0.0  10.72  -16.11
+        The three numbers are the energy of the reactant, transition state and product.
+        The barrier height for barrierless reactions is given as max(Ereact,Eprod)
+        """
+
+        with open('reaction_smi.out', 'w') as f:
```

Comment:
Could it sometimes happen that two pesviewers are called in the same directory? If so, the reaction_smi.out will be overwritten. That is why I add the `options['id']` to all file names.